.gitignore

```diff
@@ -2,3 +2,4 @@
 **/*.idea
 **/*node_modules
 frontend/jinja
+package-lock.json
```

Comment:
This should be committed into source control - https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5